src/Sylius/Bundle/InstallerBundle/Resources/config/services.xml

```diff
@@ -90,6 +91,10 @@
         <service id="sylius.installer.yaml_persister" class="%sylius.installer.yaml_persister.class%">
             <argument>%kernel.root_dir%/config/parameters.yml</argument>
         </service>
+
+        <service id="sylius.installer.checker.command_directory" class="%sylius.installer.checker.command_directory.class%">
```

Comment:
Not sure about the service name. Will we have more checkers? (see comment above about checker name)